openrouteservice-api-tests/src/test/java/heigit/ors/v2/services/routing/ParamsTest.java

```diff
@@ -1338,4 +1338,110 @@ public void expectSkipSegmentsWarnings() {
                 .body("features[0].properties.warnings[0].containsKey('message')", is(true))
                 .statusCode(200);
     }
+
+	/**
+	 * This test needs the maximum_snapping_radius of cycling-regular to be >= 10 and <= 100
+     * and maximum_snapping_radius of the default parameters to be >= 350 in the test config to run through.
+	 */
+	@Test
+	public void expectPointNotFoundError() {
+
+        JSONArray coords = new JSONArray();
+        coords.put(new JSONArray(new double[]{8.688544, 49.435462}));
+        coords.put(new JSONArray(new double[]{8.678727, 49.440115}));
+
+        JSONObject body = new JSONObject();
+        body.put("coordinates", coords);
+
+        given()
+                .header("Accept", "application/json")
+                .header("Content-Type", "application/json")
+                .pathParam("profile", "cycling-mountain")
+                .body(body.toString())
+                .when().log().all()
```

Comment:
For the tests, it's better to put `.when().log().ifValidationFails()` and `.then().log().ifValidationFails()` as that only prints the output when the test fails. It's not a problem when running through IntelliJ, but on the Travis etc. it is a nightmare to find the information if everything is printing out the full response.